pyfar/signals/deterministic.py

```diff
@@ -166,9 +161,11 @@ def linear_sweep(n_samples, frequency_range, n_fade_out=90, amplitude=1,
 
     Returns
     -------
-    sweep : pyfar.Signal
+    sweep : Signal
         The sweep as a Signal object. The Signal is in the time domain and
-        has the 'rms' FFT normalization (see pyfar.fft.normalization).
+        has the 'rms' FFT normalization (see
```

Comment:
Why rms-normalization?

Reply:
it was my first reflex. Giving it another thought, 'PSD' might make more sense, as it would be independent of the signal duration and sampling_rate. I noted it for discussion in the weekly.